explorer/Explorer.tsx

```diff
@@ -87,18 +88,85 @@ export interface ExplorerProps extends SerializedGridProgram {
     selection?: SelectionArray
 }
 
+const grapherConfigPath = (chartId: number): string =>
+    `${ADMIN_BASE_URL}/admin/api/charts/${chartId}.config.json`
+const partialGrapherConfigPath = (variableId: number): string =>
+    `${ADMIN_BASE_URL}/admin/api/variables/${variableId}.grapherConfig.json`
+
+async function fetchJSON<T>(url: string): Promise<T> {
```

Comment:
We should probably be using the [`getJSON` function in `Admin.tsx`](https://github.com/owid/owid-grapher/blob/330ee415038ed4d233d36e2c7d7d99022065ddb6/adminSiteClient/Admin.tsx#L178), but I didn't want there to be an explicit link between the Admin and the Explorer